File: cslavb/Csla/Core/FieldManager/PropertyInfoList.vb
Imports System
Imports System.Collections.Generic

Namespace Core.FieldManager
  Friend Class PropertyInfoList
    Inherits List(Of IPropertyInfo)

    Private _IsLocked As Boolean
    Public Property IsLocked() As Boolean
      Get
        Return _IsLocked
      End Get
      Set(ByVal value As Boolean)
        _IsLocked = value
      End Set
    End Property

    Public Sub New()

    End Sub

    Public Sub New(ByVal list As List(Of IPropertyInfo))
      MyBase.New(list)
    End Sub

  End Class
End Namespace

File: cslacs/Csla/Core/FieldManager/PropertyInfoList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Csla.Core.FieldManager
{
  internal class PropertyInfoList : List<IPropertyInfo>
  {
    public bool IsLocked { get; set; }

    public PropertyInfoList()
    { }

    public PropertyInfoList(IList<IPropertyInfo> list)
      : base(list)
    { }
  }
}
